### DISCUSSION TITLE
🚀 Feature: Fehlermeldung an einer Gruppe von Checkboxen

### DISCUSSION THREAD

**Discussion by marienfeldtom** [public-ui/kolibri#6816] [Ideas]
### Goals

1. Nur eine Fehlermeldung für eine Gruppe von Checkboxen anzeigen

### Background

Wenn ich eine Gruppe von Checkboxen benutze, kann ich dieser nicht als Gruppe eine Fehlermeldung geben. Ich kann nur einzelnen Checkboxen eine Fehlermeldung geben. Es wäre schön, eine Fehlermeldung wie bei den Radios an der gesamten Gruppe anzeigen zu können (bspw. mit einem Fieldset).

![image](https://github.com/user-attachments/assets/99b0456a-d4a6-4f03-be8e-ed09a1e452dd)

### Proposal

Am besten wäre die Lösung mithilfe eines Fieldsets.

**Comment by deleonio** [comment]
Das Szenario ist nicht neu. Ich weiß jetzt nicht, ob wir hierfür schon ein Beispiel in der [Sample-App](https://public-ui.github.io/v2/sample-react) haben. Die Frage kam jedoch schon mehrfach. Es ist möglich mittels `_hideError` die Fehlermeldung zwar für die Barrierefreiheit zugänglich aber nicht sichtbar zu machen. Die sichtbare Fehlermeldung platziert man dann selbst mittels eines `KolAlert`s innerhalb eines `Fieldset`s.

**Comment by marienfeldtom** [comment]
So haben wir das auch schon probiert. Allerdings ist es dann nicht möglich, eine konsistente Lösung mit dem roten Fehlerbalken auf der linken Seite zu erzeugen (hier der Vergleich zum Radio, fehlende Icons bitte ignorieren, haben bei CodeSandbox nicht geladen).

![image](https://github.com/user-attachments/assets/69e95d51-fb92-45b2-9278-d42a5c72cace)

**Comment by deleonio** [comment]
Hallo @marienfeldtom,

wir haben im letzten Release ein Beispiel nachgezogen, wie man das mittels KoliBri umsetzen kann: https://develop--kolibri-public-ui.netlify.app/#/scenarios/input-group-with-error

Code: https://github.com/public-ui/kolibri/blob/develop/packages/samples/react/src/scenarios/input-group-with-error.tsx

**Comment by marienfeldtom** [comment]
Hi @deleonio,

ist ja so umgesetzt wie in meinem Screenshot auch. Visuell ist das dann aber nicht konsistent, wenn man das beispielsweise mit dem Radio vergleicht (der rote Balken links). Nehmt ihr das so hin oder können wir mit einer Anpassung rechnen? Sonst müssen wir die CheckBox selbst umsetzen.

**Reply by deleonio** [reply]
Hi @marienfeldtom,

im `default`-Theme ist das noch nicht korrekt umgesetzt. Hier müsste man bei einem `_hideError` auch die linke rote Line entfernen und das dann der Komposition überlassen. Hierzu werde ich ein Ticket (https://github.com/public-ui/kolibri/issues/6870) erstellen.

Habt Ihr ein eigenes Theme? Dort solltet Ihr das vornehmen, da diese Art des Stylings nicht im Basis-Styling innerhalb der Webcomponents zentral vorgenommen wird.
